Tests.Unit.Core.ApplicationServices/ApplicationServices/UserServiceTest.cs

```diff
@@ -270,6 +274,130 @@ public void DeleteUserFromKitos_Raises_EntityBeingDeletedEvent_And_AccessRights_
             transaction.Verify(x => x.Commit(), Times.Once);
         }
 
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin, false, true, false)]
+        [InlineData(OrganizationRole.LocalAdmin, false, true, false)]
+        [InlineData(OrganizationRole.GlobalAdmin, true, true, false)]
+        public void DeleteUserFromKitos_Raises_EntityBeingDeletedEvent(
```

Comment:
once you rename a method make sure the test methods that reference it are also renamed